frontends/web/src/components/forms/input.tsx

```diff
@@ -81,7 +79,7 @@ export default forwardRef<HTMLInputElement, Props>(function Input({
                 spellCheck={false}
                 type={type}
                 id={id}
-                ref={ref || inputRef}
+                ref={ref}
```

Comment:
makes sense, could you update the commit message that passing inputRef via props isn't needed with forwardRef and add this link https://reactjs.org/docs/uncontrolled-components.html#default-values